### ISSUE TITLE
CoreCLR

### ISSUE THREAD

**Issue by CumpsD** [fsprojects/fsharp-dnx#27]
Any plans to try and get this to work with the coreclr runtime?

**Comment by Alxandr** [comment]
Nope. Cause DNX is dead. The dotnet CLI is the new and shiny. My understanding is that as of RC2 there won't be a DNX anymore, thus this project will stop being useful.
